modules/hooks.nix

```diff
@@ -2894,6 +2894,13 @@ lib.escapeShellArgs (lib.concatMap (ext: [ "--ghc-opt" "-X${ext}" ]) hooks.ormol
             builtins.toString script;
           files = "\\.nix$";
         };
+      nix-fmt =
+        {
+          name = "nix-fmt";
+          description = "Nix formatter.";
```

Comment:
Lets come up with a better description here, otherwise this looks like another formatter, instead of something that calls a formatter from the flake.

Maybe something along the lines of "Format Nix files using the formatter specified in your flake."?